pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -91,8 +91,7 @@ type Service struct {
 	enqueueBlock func(*block.Block) error
 	account      *wallet.Account
 
-	oidsCh   chan oid.ID
-	blocksCh chan *block.Block
+	oidsCh chan oid.ID
```

Comment:
> Previously, the `blockQueuer` routine, which enqueues blocks into `bQueue`, could block if NeoFS returned some blocks slowly.

s/`could block`/`could be blocked on enqueing newer blocks if older blocks downloading is delayed by NeoFS`

And extend the commit message, it's important to describe the the whole reason. blocksCh channel is a queue in fact, and the order of blocks in this channel depends on blocks downloading speed, whereas block queue expects the exact range of blocks.

We need these extended commit messages for git history, it's important to make them as detailed as possible for such unobvious bugfixes, because in two months I'll forget about the reason why we did this fix.